detect_secrets/plugins/aws.py

```diff
@@ -24,13 +24,16 @@ class AWSKeyDetector(RegexBasedDetector):
 
     denylist = (
         re.compile(r'AKIA[0-9A-Z]{16}'),
+        re.compile(r'aws.{0,20}?[\'\"]([0-9a-zA-Z/+]{40})[\'\"]'),
```

Comment:
```suggestion

        # This examines the variable name to identify AWS secret tokens.
        # The order is important since we want to prefer finding `AKIA`-based
        # keys (since they can be verified), rather than the secret tokens.
        re.compile(r'aws.{0,20}?[\'\"]([0-9a-zA-Z/+]{40})[\'\"]'),
```